File: C/1180 - Menor e Posição.c
#include <stdio.h>

int main(void){

	int n, smaller = 1000000, pos, cont;

	scanf(" %d", &n);

	int vector[n];

	for (cont = 0; cont < n; cont++){
        scanf(" %d", &vector[cont]);

        if (vector[cont] < smaller){
            smaller = vector[cont];
            pos = cont;
        }
	}

	printf("Menor valor: %d\n", smaller);
	printf("Posicao: %d\n", pos);

	return 0;
}

File: Python/Lista 5/1180 - Menor e Posição.py
n = int(input())

temp = input().split(' ')

v = []

for i in range(0, n):
    v.append(int(temp[i]))

for i in range(0, n):
    if i == 0:
        menor = v[i]
        pos = i

    else:
        if v[i] < menor:
            menor = v[i]
            pos = i

print("Menor valor: {}".format(menor))
print("Posicao: {}".format(pos))
